<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Forward</class>
 <widget class="QDialog" name="Forward">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>431</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>567</width>
    <height>567</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>转发设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/new/prefix1/icon/relate.png</normaloff>:/new/prefix1/icon/relate.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_13">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_12">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_11">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_5">
             <property name="horizontalSpacing">
              <number>4</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>转发模式</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QGroupBox" name="groupBox_2">
               <property name="title">
                <string/>
               </property>
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="gridLayout_3">
                  <item row="0" column="0">
                   <widget class="QRadioButton" name="radioButton">
                    <property name="text">
                     <string>直接转发</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QRadioButton" name="radioButton_2">
                    <property name="text">
                     <string>模板转发（模板）</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_9">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_8">
               <item row="0" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>发送目标</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <layout class="QGridLayout" name="gridLayout_7">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>发送类型</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QComboBox" name="comboBox_2">
                   <item>
                    <property name="text">
                     <string>TCP</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>UDP单播</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>UDP广播</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>UDP组播</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="QPushButton" name="pushButton_3">
                   <property name="text">
                    <string>确定</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>目标地址</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="2">
                  <widget class="QPushButton" name="pushButton_4">
                   <property name="text">
                    <string>删除</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QComboBox" name="comboBox"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <widget class="QTableWidget" name="tableWidget"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>确定</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string>取消</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
